src/spdx_tools/spdx3/model/ai/ai_package.py

```diff
@@ -0,0 +1,87 @@
+#  SPDX-FileCopyrightText: 2023 spdx contributors
+#
+#  SPDX-License-Identifier: Apache-2.0
+from dataclasses import field
+from datetime import datetime
+from enum import Enum, auto
+from typing import Dict, List, Optional
+
+from spdx_tools.common.typing.type_checks import check_types_and_set_values
+from spdx_tools.spdx3.model import CreationInformation, ExternalIdentifier, ExternalReference, IntegrityMethod
+from spdx_tools.spdx3.model.software import SoftwarePurpose
+from spdx_tools.spdx.model import Package
+
+
+class SafetyRiskAssessmentType(Enum):
+    SERIOUS = auto()
+    HIGH = auto()
+    MEDIUM = auto()
+    LOW = auto()
+
+
+class AIPackage(Package):
+    energy_consumption: Optional[str] = None
+    standards_compliance: List[str] = field(default_factory=list)
+    limitations: Optional[str] = None
+    type_of_model: List[str] = field(default_factory=list)
+    information_about_training: Optional[str] = None
+    information_about_application: Optional[str] = None
+    hyperparameters: Dict[str, Optional[str]] = field(default_factory=dict)
+    data_preprocessing_steps: Optional[str] = None
+    model_explainability_mechanisms: Optional[str] = None
+    sensitive_personal_information: Optional[bool] = None
```

Comment:
The definition for `PresenceType` also lists `noAssertion`. But as far as I recall and understand the discussion on this topic this is equivalent with the property not being present, so being `None` and that is why we simply use `bool` here, correct? 
Does also apply to `Dataset` but I forgot to ask in that PR.

Reply:
Yeah, the presence type thing is not fully fleshed out, I believe. It is essentially just an optional boolean as I have implemented here. We can adapt this when it turns out to be insufficient.